index.js

```diff
@@ -72,21 +71,21 @@ module.exports = class InstallFromLink extends Plugin {
 										try {
 											files = fs.readdirSync(pj(pluginsDir, repoName))
 										} catch (e) { // readdirSync failed
-											return announce(`Cloning apparently succeeded, but the resulting installation folder was nowhere to be found. (Looking for ${pj(pluginsDir, repoName)})`) // just do this for now I guess
+											return announce(`Cloning apparently succeeded, but the resulting installation folder was nowhere to be found. (Looking for ${pj(pluginsDir, repoName)})`, "yellow") // just do this for now I guess
 										}
 										if (files.includes("manifest.json")) {
 											await powercord.pluginManager.remount(repoName)
 											if (powercord.pluginManager.plugins.has(repoName)) {
-												announce("The plugin was installed and loaded successfully!")
+												announce("The plugin was installed and loaded successfully!", "green")
 											} else { // remount mysteriously failed
-												announce("The plugin was installed, but you'll need to press Ctrl-R to reload Discord before you can use it.")
+												announce("The plugin was installed, but you'll need to press Ctrl-R to reload Discord before you can use it.", "green")
 											}
 										} else { // no manifest
-											announce("That repo is not a Powercord plugin. It's been cloned anyway, but don't expect it to do anything.")
+											announce("That repo is not a Powercord plugin. It's been cloned anyway, but don't expect it to do anything.", "orange")
 										}
 									} else { // non-zero exit code
-										if (status) announce(status)
-										else announce(`The command mysteriously exited with a non-zero status code (${code}). Do y`)
+										if (status) announce(status, "red")
+										else announce(`The command mysteriously exited with a non-zero status code (${code}). Do y`, "red")
```

Comment:
I don't know how that unfinished sentence got there. If you don't know what to do about it, just remove the "do y".